content/blog/2022/2022-12-13-dask-table.md

```diff
@@ -0,0 +1,34 @@
++++
+author = "Noah Novšak"
+date = "2022-12-13"
+draft = false
+title = "Orange you going to ask about dask"
+type = "blog"
+thumbImage = "/blog_img/2022/2022-12-13_dask_data.png"
+frontPageImage = "/blog_img/2022/2022-12-13_dask_data.png"
+blog = ["dask", "widgets"]
+shortExcerpt = "Preparing Orange to handle much more data."
+longExcerpt = "A note on using dask to deal with more and more data with Orange."
++++
+
+As great as Orange is at simplifying machine learning and bringing it closer to your [everyday regular normal guy](https://www.youtube.com/watch?v=5PsnxDQvQpw), it turns out, there are some major drawbacks that stem from it's very inception. This is because Orange is meant to be run on a laptop. A device with just a handful of CPU cores, and maybe a couple of gigabytes of RAM. Due to this ideology we have so far reasoned that nobody in their right mind would want to throw a very large dataset at it. So most of Orange's components assume they can fit all the data into memory at once, process it on a single core, and expect results fairly quickly.
+
+You might see an issue here. What happens if I give Orange more data than my poor old laptop can handle?
```

Comment:
I kinda like leaving it up to the imagination, but perhaps you can also give a hint of what happens. Your call.